imports/plugins/core/orders/client/containers/orderDashboardContainer.js

```diff
@@ -141,8 +166,40 @@ class OrderDashboardContainer extends Component {
     });
   }
 
+  // Extracted Tracker logic for the search subscription, to allow calling in both
+  // componentDidMount and componentWillReceiveProps
+  // This tracker is setup in the class itself because we need to re-subscribe when search input changes
```

Comment:
I see your comment there, but is there no way to do this in the container?

Reply:
Hmmm.... I thought so too, but couldn't come up with an approach to achieve that. I'll take another stab at it.

Reply:
Update: To be clear, we are saying we want to do this in the compose wrapper function. At the moment that code is in the OrderDashboardContainer.js file